PROJECT-BRIEF.md

```diff
@@ -1,59 +1,186 @@
+| {DEV_1} & {DEV_2}     | [8. As a user, I want to see a welcoming prompt to add my first item if my list is empty to help me get oriented to how the app works. ]({REPO_URL}/issues/8) |
+| {DEV_3} & {DEV_4}     | [9. As a user, I want to mark an item on my shopping list as purchased so the app can learn how often I buy different items.]({REPO_URL}/issues/9)            |
+
+| 📖 Learning modules                                                                           | Description |
+| --------------------------------------------------------------------------------------------- | ----------- |
+| [What to include on your Linkedin profile](./learning-modules/what-to-include-on-linkedin.md) |             |
+
+| 👋 Sync meetings | Agenda                                                                                                                                                            |
+| ---------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Weekly Sync      | - Demo <br> - Discuss Learning Modules <br> - Next week's issues                                                                                                  |
+| Pair Programming | Schedule a pairing session(s) with your partner early in the week to complete the development work required to meet the acceptance criteria of your Github issue. |
+| Optional         | Office Hours                                                                                                                                                      |
+
+### Week 5, {DATES_WEEK_5}
+
+| 🤝 Developer pairings | 🖥 Assigned Github issue                                                                                                                                                                                |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| {DEV_2} & {DEV_3}     | [10. As a user, I want to be alerted when I’m entering an empty item or an item that's already on my list.]({REPO_URL}/issues/10)                                                                       |
+| {DEV_1} & {DEV_4}     | [11. As an item, I want my estimated next purchase date to be computed at the time my purchase is recorded in the database so the app can learn how often I buy different items.]({REPO_URL}/issues/11) |
+
+| 📖 Learning modules                                                        | Description |
+| -------------------------------------------------------------------------- | ----------- |
+| [Navigating finances during a career change](https://youtu.be/xtkgOmFPUJU) |             |
```

Comment:
Possible Descriptions: In this video, you'll learn how to manage your finances as you transition into a role as a developer. 

_Bulleted points, if needed:_
You'll learn:

- how to be financially successful 
- how to create a plan
- example scenarios
- things to be aware of in Tech